build/gulpfile.sql.js

```diff
@@ -133,15 +133,25 @@ gulp.task('package-external-extensions', task.series(
 						// And now use gulp-json-editor to modify the contents
 						const updateData = JSON.parse(fs.readFileSync(vscodeManifestFullPath)); // Read in the set of values to replace from package.vscode.json
 						Object.keys(updateData).forEach(key => {
-							data[key] = updateData[key];
+							if (key !== 'contributes') {
+								data[key] = updateData[key];
+							}
 						});
-						if(data.contributes?.menus){
+						if (data.contributes?.menus) {
 							// Remove ADS-only menus. This is a subset of the menus listed in https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/api/common/menusExtensionPoint.ts
 							// More can be added to the list as needed.
 							['objectExplorer/item/context', 'dataExplorer/context', 'dashboard/toolbar'].forEach(menu => {
 								delete data.contributes.menus[menu];
 							});
 						}
+
+						// add any configuration properties from the package.vscode.json
+						if (updateData.contributes?.configuration[0]?.properties) {
```

Comment:
Add a note here that we currently only support bringing over properties in the first config object found and don't support modifying the title.